### PR TITLE
itkdev: interest period changed to config

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

n/a

#### Description

Change interestperiods to look like thresholdperiod and branchesconfig, a similar pr will be created in dpl-cms

#### Screenshot of the result

image

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

n/a

### CODE REVIEW

File: src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx [hunk 1]
```diff
@@ -95,15 +92,15 @@ const PhysicalListDetails: FC<PhysicalListDetailsProps & MaterialProps> = ({
 
   const changeExpiryDate = useCallback(
     (e: ChangeEvent<HTMLSelectElement>) => {
-      const newEpiryDate = formatInterestPeriods.filter(
+      const newEpiryDate = interstPeriods.filter(
```
Comment (yepzdk) [inline src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx:95]: Was const `newEpiryDate` meant to be `newExpiryDate` ?

**Review by yepzdk — APPROVED** [review]
Neat optimisation.
Only commented on one very little thing to consider.

**Review by spaceo — APPROVED** [review]
So much needed! Thx.
One typo though, as Jesper pointed out.

File: src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx (hunk 1 above)
Comment (sinejespersen) [reply src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx:95]: ha, 
image

I'll rename

**Pull request merged**